.eslintrc

```diff
@@ -31,6 +31,8 @@
         "template-curly-spacing": [2, "never"],
         "no-unused-vars": 2,
         "comma-dangle": 1,
-        "no-extra-semi": 1
+        "no-extra-semi": 1,
+        "no-redeclare": ["error", { "builtinGlobals": false }]
```

Comment:
Is there a reason we want `builtinGlobals` set to false?

Reply:
This is because bumping eslint caused this to default to `true` (whereas before it was defaulted to `false`). This caused the build to fail.

Reply:
Thanks @tjsilver, yes we also has the import global which we removed (thinking it was commented out).
We assume one of the various updates changed this in the `eslint` file (probably the eslint update)